opal/mca/btl/uct/btl_uct_types.h

```diff
@@ -90,6 +90,8 @@ struct mca_btl_uct_conn_req_t {
     /** transport index that should be connected */
     int tl_index;
 
+    char module_name[16];
```

Comment:
Why saving the name and using strcmp all the time instead of saving the pointer to the module itself ?